src/main/java/mops/businesslogic/permission/PermissionServiceImpl.java

```diff
@@ -51,4 +51,21 @@ public DirectoryPermissions savePermissions(DirectoryPermissions permissions) th
             throw new DatabaseException("Die Ordnerberechtigungen konnten nicht gespeichert werden!", e);
         }
     }
+
+    /**
+    * {@inheritDoc}
+    */
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    @Override
+    public void delete(Directory directory) throws MopsException {
```

Comment:
can you rename this to deletePermissions because otherwise it looks like you are deleting the folder ;)